### PR TITLE
Test with Jekyll 2 and 3, remove legacy opal support

### PR DESCRIPTION
**Pull request by Stargator**
Based of the discussion at PR #8.

I just need to double check whether tilt is needed.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
 gemspec
+
+gem "tilt"
```
Comment (elia) [inline Gemfile]: I think this should go into the gem spec as a runtime dependency

**Comment by parkr** [comment]
@elia I was able to get everything to pass in #8 without the inclusion of Tilt. In the Gemfile.lock on my local machine, I see:

```text
 33     opal (0.8.1)
 34       hike (~> 1.2)
 35       sourcemap (~> 0.1.0)
 36       sprockets (~> 3.1)
 37       tilt (>= 1.4)
```

**Comment by elia** [comment]
@parkr even better, btw just checked and it's still there in opal-0.9 (unreleased) so thing should be fine as they are for a while :)

**Comment by Stargator** [comment]
@parkr I determined that as well.

**Comment by parkr** [comment]
Thanks so much, @Stargator and @elia!

**Comment by elia** [comment]
:)

**Comment by parkr** [comment]
Just pushed v0.2.0 with all these changes. :)

**Comment by Stargator** [comment]
@parkr YAY!

**Pull request closed without merging**